### PR TITLE
[REF] auto-fix conventions

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

**Comment by eLBati** [comment]
:+1:

File: __unported__/partner_address_ldap/address.py [hunk 1]
```diff
@@ -51,6 +51,7 @@
 
 
 class LdapConnMApper(object):
+
     """LdapConnMApper: push specific fields from the Terp
     Partner_contacts to the LDAP schema inetOrgPerson. Ldap bind options
     are stored in company.
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/address.py:56]: :-1:

File: __unported__/partner_address_ldap/address.py [hunk 2]
```diff
@@ -114,6 +115,7 @@ def get_connexion(self):
 
 
 class LDAPAddress(orm.Model):
+
     """Override the CRUD of the objet in order to dynamically bind to ldap"""
     _inherit = 'res.partner.address'
     ldapMapper = None
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/address.py:119]: :-1:

File: __unported__/partner_address_ldap/company.py [hunk 3]
```diff
@@ -36,6 +36,7 @@
 
 
 class Res_company(orm.Model):
+
     """Define ldap connexion parameters"""
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/company.py:39]: :-1:

File: __unported__/partner_address_ldap/partner.py [hunk 4]
```diff
@@ -36,6 +36,7 @@
 
 
 class LdapPartner(orm.Model):
+
     """Ensure that when deleting a partner unlink function is called on all
     related addresses"""
```
Comment (bwrsandman) [inline __unported__/partner_address_ldap/partner.py:40]: :-

File: base_location/better_zip.py [hunk 5]
```diff
@@ -23,6 +23,7 @@
 
 
 class BetterZip(orm.Model):
+
     " City/locations completion object"
```
Comment (bwrsandman) [inline base_location/better_zip.py:26]: :-

File: base_partner_merge/base_partner_merge.py [hunk 6]
```diff
@@ -81,6 +81,7 @@ class MergePartnerLine(orm.TransientModel):
 
 
 class MergePartnerAutomatic(orm.TransientModel):
+
     """
     The idea behind this wizard is to create a list of potential partners to
     merge. We use two objects, the first one is the wizard for the end-user.
```
Comment (bwrsandman) [inline base_partner_merge/base_partner_merge.py:86]: :-

File: base_partner_sequence/partner.py [hunk 7]
```diff
@@ -25,6 +25,7 @@
 
 
 class ResPartner(orm.Model):
+
     """Assigns 'ref' from a sequence on creation and copying"""
```
Comment (bwrsandman) [inline base_partner_sequence/partner.py:28]: :-

File: partner_firstname/partner.py [hunk 8]
```diff
@@ -22,6 +22,7 @@
 
 
 class ResPartner(orm.Model):
+
     """Adds lastname and firstname, name become a stored function field"""
```
Comment (bwrsandman) [inline partner_firstname/partner.py:25]: :-1:

File: passport/tests/test_passport.py [hunk 9]
```diff
@@ -26,6 +26,7 @@
 
 
 class Base_Test_passport(TransactionCase):
+
     """Simple test creating a passport
     This is a base class for passport test cases.
```
Comment (bwrsandman) [inline passport/tests/test_passport.py:30]: :-1:

File: passport/tests/test_passport.py [hunk 10]
```diff
@@ -75,6 +76,7 @@ def test_passport(self):
 
 
 class Test_passport_bad(Base_Test_passport):
+
     """Simple test creating a passport, test against bad values"""
```
Comment (bwrsandman) [inline passport/tests/test_passport.py:78]: :-1:

File: passport/tests/test_passport.py [hunk 11]
```diff
@@ -113,6 +115,7 @@ def test_passport(self):
 
 
 class Test_passport_name_get(TransactionCase):
+
     """Test name_get"""
```
Comment (bwrsandman) [inline passport/tests/test_passport.py:116]: :-1:

File: res_partner_affiliate/res_partner.py [hunk 12]
```diff
@@ -23,6 +23,7 @@
 
 
 class ResPartner(orm.Model):
+
     """
     Add relation affiliate_ids
```
Comment (bwrsandman) [inline res_partner_affiliate/res_partner.py:27]: :-1:

**Comment by bwrsandman** [comment]
Fixed up auto correct issues

**Comment by hbrunn** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**